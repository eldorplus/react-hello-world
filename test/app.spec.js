import React from 'react';
import App from '../src/app';
import renderer from 'react-test-renderer';

test('App renders components', () => {
  const component = renderer.create(
    <App />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
