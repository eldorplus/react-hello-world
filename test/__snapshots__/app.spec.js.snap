exports[`test App renders components 1`] = `
<div>
  <div>
    <h2>
      Example 1 - 
      <small>
        click once to toggle
      </small>
    </h2>
    <input
      onClick={[Function]}
      type="submit"
      value="Toggle message" />
    <p>
      Hello world!!!
    </p>
  </div>
  <div>
    <h2>
      Example 2 - 
      <small>
        click 
        3
         times to toggle
      </small>
    </h2>
    <input
      onClick={[Function]}
      type="submit"
      value="Toggle message" />
    <p>
      Hello world!!!
    </p>
  </div>
</div>
`;
